packages/round-manager/src/features/round/ViewApplicationPage.tsx

```diff
@@ -250,13 +311,15 @@ export default function ViewApplicationPage() {
               <hr className="my-6"/>
 
               <h2 className="text-xs mb-2">Funding Sources</h2>
-              <p className="text-base mb-6">{ getAnswer("Funding Source", application) }</p>
+              <p className="text-base mb-6">{ getAnswer("Funding Source") }</p>
 
               <h2 className="text-xs mb-2">Funding Profit</h2>
-              <p className="text-base mb-6">{ getAnswer("Profit2022", application) }</p>
+              <p className="text-base mb-6">{ getAnswer("Profit2022") }</p>
 
               <h2 className="text-xs mb-2">Team Size</h2>
-              <p className="text-base mb-6">{ getAnswer("Team Size", application) }</p>
+              <p className="text-base mb-6">{ getAnswer("Team Size") }</p>
+
+              <p className="text-base mb-6">{getAnswer("Is your project cool enough to apply ?")}</p>
```

Comment:
is this a real application question? o.0

Reply:
nah it's not! it should be removed

Reply:
```suggestion
```